frontend/src/exams/edit-exam-form-modal.vue

```diff
@@ -301,14 +298,12 @@
         return ''
       },
       examType() {
-        if (this.examRow && this.examRow.exam_type) {
```

Comment:
Was `this.examRow` removed in a previous PR or something? I don't see it anywhere in this file.

Reply:
examRow was turned into exam a long time ago so this function has been else'ing for a long time